src/utils.ts

```diff
@@ -92,7 +92,9 @@ export function todayInMinutes() {
   return today.diff(dayjs().startOf('day'), 'minute')
 }
 
-export function modeToNum(mode: Mode, current?: dayjs.Dayjs | Date): number {
+export function modeToNum(mode: Mode, current?: dayjs.Dayjs | Date, direction?: String): number {
+  const isLeft = direction === 'LEFT'
```

Comment:
- `isRtl` is more explicit than `isLeft`
- I think this is incorrect because the unit test fails

```suggestion
  const isRtl = direction === 'LEFT'
```

Reply:
Please take a look at the CI result:

https://github.com/acro5piano/react-native-big-calendar/actions/runs/3807586563/jobs/6480860145